tests/test_providers.py

```diff
@@ -123,6 +123,11 @@ def test_get_by_uri(self):
         self.assertEqual(concept['uri'], 'http://vocab.getty.edu/aat/300007466')
         self.assertEqual(concept['id'], '300007466')
 
+    def test_get_by_uri_invalid(self):
```

Comment:
Could you add a test on the URI https://id.erfgoed.net/thesauri/materialen/7 as well? Should be False as well.